src/graph/graphcontext.c

```diff
@@ -27,6 +27,23 @@ extern RedisModuleType *GraphContextRedisModuleType;
 static void _GraphContext_Free(void *arg);
 static void _GraphContext_UpdateVersion(GraphContext *gc, const char *str);
 
+static uint64_t _count_indices_from_schemas(const Schema** schemas) {
+	ASSERT(schemas);
+	uint64_t count = 0;
+
+	const uint32_t length = array_len(schemas);
+	for (uint32_t i = 0; i < length; ++i) {
```

Comment:
as a general rule, we use suffix addition unless there's a good reason for performing prefix addition

Reply:
I wasn't aware of that, looking at Redis source code most of the for loops I could find are using postfix increment.

Reply:
Appreciate these insights.